### ISSUE TITLE
OPM on pre-built Win32 package

### ISSUE THREAD

**Issue by akaegi** [openresty/opm#53]
Hello,
I cannot find OPM in the pre-built Win32 package? Am I missing something, can I install it manually or is OPM not supported on Windows?

**Comment by agentzh** [comment]
@akaegi I don't think it is supported on Windows yet. But you're welcome to test and contribute fixes if there's any problems. Thanks!
